src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -711,6 +711,9 @@ public async Task Delete()
                             if (rawRecord == null)
                                 continue;
 
+                            if (rawRecord.DatabaseState == DatabaseStateStatus.RestoreInProgress)
+                                throw new InvalidOperationException($"Can't delete database {databaseName} while restore is in progress.");
```

Comment:
```suggestion
                                throw new InvalidOperationException($"Can't delete database '{databaseName}' while the restore process is in progress. In order to delete the database, you can cancel it from node {node}.");
```
You should be able to get the `node` from the topology.

Reply:
done